src/components/WorkspaceDetails/WorkspaceDetails.tsx

```diff
@@ -80,11 +99,20 @@ const WorkspaceDetails = () => {
 
   return (
     <PageContentWrapper>
+      <WorkspaceDeleteModal
+        workspaceName={workspace?.metadata?.name}
+        isOpen={deleteModalOpen}
+        closeModal={() => setDeleteModalOpen(false)}
+        onDelete={() => {
+          navigate('/workspaces');
```

Comment:
NIT - maybe we move this to a constant? Murphys law says it will change out from under us :)